Controllers/FamilyController.cs

```diff
@@ -140,6 +151,15 @@ public async Task<ActionResult> CreateFamily([FromForm] FamilyViewModel Family)
 
         _jobService.CreateorUpdateEmailJob(created.Data!.Id);
 
+        // if they are eligible, add them to the letter queue
+        if (
+            NewFamily.UserStatus == UserStatus.ExtendedFreeTrial
+            || NewFamily.UserStatus == UserStatus.Subscribed
+        )
+        {
+            _letterQueueService.PopulateLetterQueue(familyId);
```

Comment:
To populate the letter queue we need the following properties from family: name, userstatus, phone number, email, family size. We also need a list of member names in the family, and their address. instead of just passing their family ID here and fetching all needed info in the letter queue again, we actually have access to most of this info before we call PopulateLetterQueue. We could save ourselves from having to hit the DB more than needed if we were to just pass the info we have available to the letter queue so that we don't have to fetch it there

Reply:
Good call, I will pass in the object so we can avoid a lot of the calls